src/cbc_sdk/cache/lru.py

```diff
@@ -96,9 +100,18 @@ class LRUCacheDict(object):
     set to true. Note that the cache will always be concurrent if a background cleanup thread
     is used.
     """
-    def __init__(self, max_size=1024, expiration=15 * 60, thread_clear=False, thread_clear_min_check=60, concurrent=True):
+
+    def __init__(self, max_size=1024, expiration=15 * 60, thread_clear=False, thread_clear_min_check=60,
+                 concurrent=True):
         """
-        Initialize LRUCacheDict
+        Initialize the LRUCacheDict object.
+
+        Args:
+            max_size (int): Maximum number of elements in the cache.
+            expiration (int): Number of seconds an item can be in the cache before it expires.
+            thread_clear (bool): True if we want to use a background thread to keep the cache clear.
+            thread_clear_min_check (int): Unused.
```

Comment:
Since `thread_clear_min_check` is unused, we should remove it

Reply:
I can remove it from the doc but I'm hesitant to remove it from the parameters list just in case it breaks something.

Reply:
Actually, turns out I had to.  Flake8 won't let me just take it out of the docstring, it complains.